packages/millicast-sdk/src/workers/TransformWorker.worker.js

```diff
@@ -10,15 +10,17 @@ const synchronizationSources = {}
 let synchronizationSourcesWithMetadata = []
 
 function createReceiverTransform (mid) {
+  const closedCaptionReady = (startTime, endTime, text) => {
+    self.postMessage({ mid, event: 'closedCaption', startTime, endTime, text })
+  }
+  resetCodecs()
```

Comment:
question: (non-blocking) @vincentsong Is this resetting the codecs after each group of CC is received, or is this running less often?